JobManager-Branch-BoundKnapsack/.idea/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We need to remove the ".idea" folder and all its content from this PR.

Reply:
Removed